src/praveen/cityBasedGroup.ts

```diff
@@ -0,0 +1,48 @@
+interface Person {
+  _id: string;
+  name: string;
+  gender: string;
+  email: string;
+  phone: string;
+  city: string;
+}
+
+interface PersonGroup {
+  count: number;
+  city: string;
+  people: Array<Person>;
+}
```

Comment:
Can we maintain separate directory for interfaces ?

Reply:
Unless if it is a common thing that shall be used at multiple places inside application, I don't find a reason to move them in to a different directory.

Reply:
Agreed with Raj on this

Reply:
I also agreed with Raj on this.